### PR TITLE
Fix : #8430 

### PR DESCRIPTION
**Pull request by lgaches**
### Checklist
- [X] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [X] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [X] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [X] I've updated the documentation if necessary.

### CODE REVIEW

**Review by giginet — APPROVED** [review]

**Comment by mfurtak** [comment]
Will it break anyone's existing workflow to make this change?

File: sigh/lib/sigh/runner.rb [hunk 1]
```diff
@@ -232,7 +232,7 @@ def certificate_to_use
     # Downloads and stores the provisioning profile
     def download_profile(profile)
       UI.important "Downloading provisioning profile..."
-      profile_name ||= "#{profile_type.pretty_type}_#{Sigh.config[:app_identifier]}"
+      profile_name ||= "#{profile_type.pretty_type}_#{profile.uuid}_#{Sigh.config[:app_identifier]}"
```
Comment (KrauseFx) [inline sigh/lib/sigh/runner.rb]: This will break setups where the user assumes the file name

File: sigh/spec/manager_spec.rb [hunk 2]
```diff
@@ -11,7 +11,7 @@
       Sigh.config = FastlaneCore::Configuration.create(Sigh::Options.available_options, options)
 
       val = Sigh::Manager.start
-      expect(val).to eq(File.expand_path("./AppStore_com.krausefx.app.mobileprovision"))
+      expect(val).to eq(File.expand_path("./AppStore_uuid_com.krausefx.app.mobileprovision"))
```
Comment (KrauseFx) [inline sigh/spec/manager_spec.rb]: Yes, that's why this change would break existing setups. These tests must keep working without them being changed.

**Review by KrauseFx — CHANGES_REQUESTED** [review]

**Comment by KrauseFx** [comment]
Any updates on this?

**Comment by KrauseFx** [comment]
I'll go ahead and close this PR for now. Please feel free to submit a new issue, so we can continue the discussion 👍

**Pull request closed without merging**